ENIGMAsystem/SHELL/Widget_Systems/Win32/dialogs.cpp

```diff
@@ -287,12 +287,12 @@ void show_info(string info, int bgcolor, int left, int top, int width, int heigh
     if (showBorder) {
       flags |= WS_BORDER | WS_DLGFRAME | WS_CAPTION;
     }
-    if (stayOnTop) {
-      flags |= DS_MODALFRAME; // Same as WS_EX_TOPMOST
-    }
     if (allowResize) {
       flags |= WS_SIZEBOX;
     }
+    if (stayOnTop) {
+      flags |= DS_MODALFRAME; // Same as WS_EX_TOPMOST
```

Comment:
In light of what we discovered about GWL_EXSTYLE vs GWL_STYLE, this line I wrote might not have originally been correct. I do recall difficulties in implementing the game info window to behave even remotely as expected.